tests/integration-tests/processors/processors-test.js

```diff
@@ -9,20 +9,58 @@ describe('Processors api', function() {
         await requestSender.init();
     });
 
-    describe('Good requests', function() {
+    describe('Good requests', async function() {
+        describe('GET /v1/processors', async function () {
+            let jsProcessorsArr;
+            let processorsInserted;
+            before(async function() {
+                jsProcessorsArr = [...(new Array(101).keys())].map(i => generateRawJSProcessor(i.toString()));
+                processorsInserted = await Promise.all(jsProcessorsArr.map(processor => requestSender.createProcessor(processor, validHeaders)));
+            });
+            it('Check default paging values (from = 0, limit = 100)', async function() {
+                let getProcessorsResponse = await requestSender.getProcessors();
+
+                should(getProcessorsResponse.statusCode).equal(200);
+
+                const processors = getProcessorsResponse.body;
+                should(processors.length).equal(100);
+            });
+
+            it('Get a page', async function() {
+                const from = 25, limit = 50;
+                let getProcessorsResponse = await requestSender.getProcessors(from, limit);
+
```

Comment:
i would add to this test  another `requestSender.getProcessors(from, limit);` with `from=0` and compare the two arrays' object in `getProcessorsResponse[0]` and make sure that the objects are different (this will ensure it starts at the correct `from` index)

Reply:
new test added to ensure from starts from the correct index.
does the test satisfies it? or did I miss something?